app/components/TheShareGrid.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import { useClipboard } from '@vueuse/core'
+import type { PropType } from 'vue'
+import type { Mlm } from '../types'
+import { isTest } from '~/helpers'
+
+defineProps({
+  mlm: {
+    type: Object as PropType<Mlm>,
+    required: true,
+  },
+})
+
+const { copy, isSupported } = useClipboard()
+const currentLocation = ((process.client || isTest) && location) ? location.toString() : ''
+const showIsCopied = ref(false)
+function copyToClipboard() {
+  copy(currentLocation)
+  showIsCopied.value = true
+  setTimeout(() => showIsCopied.value = false, 2000)
+}
+</script>
+
+<template>
+  <p class="my-4">
+    Share results via:
+  </p>
+  <a-row type="flex" justify="center" class="gap-10">
+    <a-col :span="8">
+      <a-button class="ShareButton ButtonPrimary" size="large" :href="`mailto:?subject=${mlm.name} is an MLM!&body=${currentLocation}`">
+        E-mail
+      </a-button>
+    </a-col>
+    <a-col v-if="isSupported" :span="8">
+      <a-button
+        class="ShareButton ButtonInfo" size="large" data-testid="copyButton" @click="copyToClipboard"
+      >
+        <span v-if="!showIsCopied">Copy link</span>
+        <span v-else class="items-center justify-center" style="display: flex">
+          <CheckOutlined />
```

Comment:
Do we want to use a margin here instead of the `&nbsp;`?

```suggestion
          <CheckOutlined class="mr-1" />
```